index.js

```diff
@@ -641,6 +641,7 @@ class GithubScm extends Scm {
         const type = payloadHeaders['x-github-event'];
         const hookId = payloadHeaders['x-github-delivery'];
         const checkoutUrl = hoek.reach(webhookPayload, 'repository.ssh_url');
+        const scmContexts = this._getScmContexts();
```

Comment:
`_getScmContexts()` is asynchronous processing, so it seems that `scmContexts` can not be acquired.